matsim-jerusalem/src/jerusalem/scenario/network/CreateNetwork.java

```diff
@@ -8,71 +8,77 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Iterator;
+import java.util.LinkedHashSet;
 import java.util.Map;
 import java.util.Set;
 import java.util.TreeMap;
 
 import org.apache.log4j.Logger;
 import org.matsim.api.core.v01.Coord;
 import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.TransportMode;
 import org.matsim.api.core.v01.network.Link;
 import org.matsim.api.core.v01.network.Network;
 import org.matsim.api.core.v01.network.NetworkFactory;
 import org.matsim.api.core.v01.network.Node;
 import org.matsim.core.network.NetworkUtils;
+import org.matsim.core.network.algorithms.MultimodalNetworkCleaner;
+import org.matsim.core.network.algorithms.NetworkCleaner;
+import org.matsim.core.network.io.MatsimNetworkReader;
 import org.matsim.core.network.io.NetworkWriter;
 
-
 /**
  * @author Golan Ben-Dor
  */
 public class CreateNetwork
 {
-	//Logger
+	// Logger
 	private static final Logger log = Logger.getLogger(CreateNetwork.class);
 
 	private final static String INPUT_LINKS_CSV = "E:\\Golan\\Jerusalem MATSim\\Data recieved\\Current State\\network_pt\\2015\\links.csv";
 	private final static String INPUT_NODES_CSV = "E:\\Golan\\Jerusalem MATSim\\Data recieved\\Current State\\network_pt\\2015\\nodes.csv";
 	private final static String OUTPUT_NETWORK_FOLDER = "E:\\Golan\\Jerusalem MATSim\\MATSim Input\\";
-	private final static boolean REMOVE_CONNECTOR = true;
+	private final static String NETWORK_ID = "" + 4;
```

Comment:
what is the purpose of the network_id?

Reply:
Also, I had different outputs (I have each run in one table). For example, one without connector, another not cleaned (nodes which are not connected to anything because of connectors removal)